subsys/net/lib/fota_download/src/fota_download.c

```diff
@@ -337,6 +346,8 @@ int fota_download_start(const char *host, const char *file, int sec_tag,
 		return err;
 	}
 
+	download_started = true;
```

Comment:
Won't we get an error from `download_client_start` if a download has already started? I guess not since you open this PR, but just double checking :)

Reply:
We will. But you have this delayed start.